Extension/src/LanguageServer/client.ts

```diff
@@ -705,12 +705,11 @@ class DefaultClient implements Client {
     }
 
     private updateInactiveRegions(params: InactiveRegionParams): void {
-        let renderOptions: vscode.DecorationRenderOptions = {
-            light: { color: "rgba(175,175,175,1.0)" },
-            dark: { color: "rgba(155,155,155,1.0)" },
-            rangeBehavior: vscode.DecorationRangeBehavior.ClosedOpen
-        };
-        let decoration: vscode.TextEditorDecorationType = vscode.window.createTextEditorDecorationType(renderOptions);
+        let settings: CppSettings = new CppSettings(this.RootUri);
+        
+        let decoration: vscode.TextEditorDecorationType = vscode.window.createTextEditorDecorationType({
+            opacity: String(settings.inactiveRegionOpacity)
```

Comment:
Do we need any error validation here?

Reply:
We don't do it elsewhere in the codebase. I believe the line of thought is ```CppSettings``` should always be constructible, as ```this.RootUri``` should always be valid. @bobbrow?

Reply:
I think Andrew is referring to the inactiveRegionOpacity setting